canal/sync.go

```diff
@@ -103,7 +106,7 @@ func (c *Canal) runSyncBinlog() error {
 				return errors.Trace(err)
 			}
 
-			c.master.UpdateGTID(gtid)
+			c.master.UpdateGTIDSet(gtid)
```

Comment:
why update GTID set here? a bit strange, this gtid is a single GTID, and if you want do it for some reasons, you don't do it in `*replication.GTIDEvent`

Reply:
If this is a single gtid, then I should remove it.